airflow/www/templates/airflow/tree.html

```diff
@@ -85,8 +85,59 @@
 <script>
 $('span.status_square').tooltip({html: true});
 
+function ts_to_dtstr(ts) {
+  var dt = new Date(ts * 1000);
+  return dt.toISOString();
+}
+
+function is_dag_run(d) {
+  return d.run_id !== undefined;
+}
+
+var now_ts = Date.now()/1000;
+
+function backfill_taskinstance_properties(node) {
+  // backfill task instance properties for display purpose
```

Comment:
Can we use a different word than backfill here - this might get confused with an `airflow backfill` but it's something else here.

Perhaps populate? Or just fill?

Reply:
@houqp ^^

Reply:
@ashb this has already been updated to populate.

Reply:
sorry, i missed the comment :(